spec/pb_kits/playbook/kits/time_range_inline_spec.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require_relative "../../../../app/pb_kits/playbook/pb_time_range_inline/time_range_inline"
+
+RSpec.describe Playbook::PbTimeRangeInline::TimeRangeInline do
+  subject { Playbook::PbTimeRangeInline::TimeRangeInline }
+
+  it { is_expected.to define_partial }
+
+  # Do not leave this file blank. Use other spec files for example tests.
```

Comment:
You will want to add your test coverage here

Reply:
Included test coverage, thanks.